modules/compute/gke-node-pool/main.tf

```diff
@@ -35,6 +35,10 @@ locals {
   module_unique_id = replace(lower(var.internal_ghpc_module_id), "/[^a-z0-9\\-]/", "")
 }
 
+data "google_container_cluster" "gke_cluster" {
+  name = split("/", var.cluster_id)[5]
```

Comment:
Please create a local variable that describes the spilited text better. something like: `cluster_name = split("/", var.cluster_id)[5]`